src/eve-server/manufacturing/FactoryDB.cpp

```diff
@@ -355,14 +358,29 @@ PyRep *FactoryDB::AssemblyLinesSelectCorporation(const uint32 corpID) {
 PyRep *FactoryDB::AssemblyLinesSelectAlliance(const int32 allianceID) {
     DBQueryResult res;
 
+    // This produces the same output but with less complex joins and
+    // the quantity column is completely redundant.
+    // SELECT
+    //     job.containerId,
+    //     station.stationTypeId AS containerTypeId,
+    //     station.solarSystemId AS containerLocationId,
+    //     job.typeId as assemblyLineTypeId,
+    //     COUNT(job.containerId) as quantity,
+    //     station.corporationId as ownerId
+    //     FROM industrySlots AS job 
+    //         JOIN evemu.staStations AS station ON station.stationId = job.containerId
+    //         JOIN evemu.crpCorporation AS corp ON corp.corporationId = station.corporationId
+    //     GROUP BY job.containerId, job.typeId;
+
     if (!sDatabase.RunQuery(res,
-        "SELECT"
+        "SELECT DISTINCT"
         " station.stationID AS containerID,"
         " station.stationTypeID AS containerTypeID,"
         " station.solarSystemID AS containerLocationID,"
         " station.assemblyLineTypeID,"
         " station.quantity,"
-        " station.ownerID"
+        " station.ownerID,"
+        " line.activityID"
```

Comment:
This was missing from all but the Public filter.  Honestly, I suspect all of the filter options can be easily combined into a single query to reduce the risk of deviating changes like this.